azure-mgmt-website/src/main/java/com/microsoft/azure/management/website/CountryISOCode.java

```diff
@@ -0,0 +1,292 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ */
+
+package com.microsoft.azure.management.website;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/**
+ * Defines values for VirtualMachineSizeTypes.
+ */
+public final class CountryISOCode {
+    // CHECKSTYLE IGNORE Javadoc FOR NEXT 237 LINES
+    public static final CountryISOCode Afghanistan = new CountryISOCode("AF");
```

Comment:
conventions - all caps for enum constants (or static public fields) -  AFGHANISTAN, ALBANIA, etc.